tests/waku_archive/test_driver_postgres.nim

```diff
@@ -86,74 +72,64 @@ suite "Postgres driver":
 
     let msg1 = fakeWakuMessage(contentTopic = contentTopic1)
 
-    var putRes = await driver.put(
-      pubsubTopic1,
-      msg1,
-      computeDigest(msg1),
-      computeMessageHash(pubsubTopic1, msg1),
-      msg1.timestamp,
-    )
+    var putRes =
+      await driver.put(computeMessageHash(pubsubTopic1, msg1), pubsubTopic1, msg1)
     assert putRes.isOk(), putRes.error
 
     let msg2 = fakeWakuMessage(contentTopic = contentTopic2)
 
-    putRes = await driver.put(
-      pubsubTopic2,
-      msg2,
-      computeDigest(msg2),
-      computeMessageHash(pubsubTopic2, msg2),
-      msg2.timestamp,
-    )
+    putRes =
+      await driver.put(computeMessageHash(pubsubTopic2, msg2), pubsubTopic2, msg2)
     assert putRes.isOk(), putRes.error
 
     let countMessagesRes = await driver.getMessagesCount()
 
     assert countMessagesRes.isOk(), $countMessagesRes.error
     assert countMessagesRes.get() == 2
 
-    var messagesRes = await driver.getMessages(contentTopic = @[contentTopic1])
+    var messagesRes = await driver.getMessages(contentTopics = @[contentTopic1])
 
     assert messagesRes.isOk(), $messagesRes.error
     assert messagesRes.get().len == 1
 
     # Get both content topics, check ordering
     messagesRes =
-      await driver.getMessages(contentTopic = @[contentTopic1, contentTopic2])
+      await driver.getMessages(contentTopics = @[contentTopic1, contentTopic2])
     assert messagesRes.isOk(), messagesRes.error
 
     assert messagesRes.get().len == 2
-    assert messagesRes.get()[0][1].contentTopic == contentTopic1
+    assert messagesRes.get()[0][2].contentTopic == contentTopic1
```

Comment:
For a better understanding, why does the index change?

Reply:
The result of archive is now (hash, topic, msg) before it was (topic, msg, digest, sender timestmap, hash)